data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelProtoCodec.java

```diff
@@ -1202,6 +1103,9 @@ public static Map<String, Object> getInstrumentationScopeAttributes(final Instru
         if (!instrumentationScope.getVersion().isEmpty()) {
             instrumentationScopeAttr.put(INSTRUMENTATION_SCOPE_VERSION, instrumentationScope.getVersion());
         }
+        if (!instrumentationScope.getAttributesList().isEmpty()) {
+            instrumentationScopeAttr.put(ATTRIBUTES_KEY, OTelProtoCodec.unpackKeyValueListLog(instrumentationScope.getAttributesList()));
```

Comment:
I think, the attributes should be put under `instumentationsScope.attributes` and filtered similar to all other attributes: Prefix + Dedotting. Won't this proposed handling add them in a very confusing location and just named `attributes`?

Reply:
I am comparing with the output generated by OTEL and it looks like they are putting it under "scope" like this 
```
"scope": {
        "name": "my.library",
        "attributes": {
          "my.scope.attribute": "some scope attribute"
        },
        "version": "1.0.0"
      },
```
So, it matches with it.